.github/workflows/build.yml

```diff
@@ -23,14 +23,19 @@ jobs:
       IsReleaseBuild: ${{ matrix.configuration == 'Release' && github.event_name == 'release' && !github.event.release.prerelease && github.ref == 'refs/heads/master' }}
 
     steps:
-    - uses: actions/checkout@v4
-    - uses: actions/setup-dotnet@v4
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Install .NET 8 SDK
+      uses: actions/setup-dotnet@v4
       with:
         dotnet-version: '8.0.x'
-    # Install SF SDK
-    - name: Install SF
+    - name: Install .NET 9 SDK
+      uses: actions/setup-dotnet@v4
+      with:
```

Comment:
Feel free to ignore but note that the two setup-dotnet@v4 actions can be combined to form a "Install .NET SDKs" step.
Syntax should be like:
dotnet-version: | 
      8.x
      9.x
https://github.com/actions/setup-dotnet